app/src/main/java/org/wikipedia/search/SearchResultsViewModel.kt

```diff
@@ -26,20 +26,20 @@ class SearchResultsViewModel : ViewModel() {
 
     private val batchSize = 20
     private val delayMillis = 200L
-    var resultsCount = mutableListOf<Int>()
+    var resultPairList = mutableListOf<Pair<String, Int>>()
```

Comment:
Can we give this a clearer name, such as `countsPerLanguageCode`?

Reply:
Done!